caravel/__init__.py

```diff
@@ -30,6 +27,18 @@
 
 migrate = Migrate(app, db, directory=APP_DIR + "/migrations")
 
+# Logging configuration
+logging.basicConfig(format='%(asctime)s:%(levelname)s:%(name)s:%(message)s')
+logging.getLogger().setLevel(logging.DEBUG)
```

Comment:
if only two things was to get parameterized it should be the logging level and the format string